lms/templates/certificates/server-error.html

```diff
@@ -0,0 +1,25 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%inherit file="../main.html" />
+
```

Comment:
@griffresch can you confirm the messaging on this template to ensure we have everything correct?